.github/workflows/ci-pre-commit.yml

```diff
@@ -4,8 +4,21 @@ on: [push, pull_request]
 
 jobs:
   build:
+    name: "Linux - pre-commit checks - Python 3.10"
+    timeout-minutes: 30
     runs-on: ubuntu-latest
+    env:
+      PRE_COMMIT_USE_MICROMAMBA: 1
     steps:
+    - name: Checkout branch
     - uses: actions/checkout@v3
-    - uses: actions/setup-python@v4
+    - name: Set up micromamba
+      uses: mamba-org/setup-micromamba@2b72821d5ad7f6da3c003a3684ce341bf187b46f
+    - name: Add micromamba to GITHUB_PATH
+      run: echo "${HOME}/micromamba-bin" >> "$GITHUB_PATH"
+    - name: Install Python 3.10
+      uses: actions/setup-python@v4
+      with:
+        python-version: "3.10"
```

Comment:
Other parts of the `plateau` CI use python 3.11 - any reason I shouldn’t update this to do the same?

Reply:
The version here really doesn't matter (feel free to use whatever feels most consistent). It's just the version for `pre-commit`. Each hook runs in a separate environment anyways.

Reply:
I just checked out the `setup-python` documentation at it seems in their docs they're still using 3.10. I doubt there's any significance to that beyond not having updated their docs, but I'll leave it as is for now